ReactiveCocoa/NSObject+Association.swift

```diff
@@ -1,5 +1,29 @@
 import ReactiveSwift
 
+internal struct AssociationKey {
```

Comment:
These seem mostly unrelated, so I think it actually makes sense to separate these into different `fileprivate` keys.